src/actions/adaptive-container.ts

```diff
@@ -28,6 +30,44 @@ export class AdaptiveActionContainer<T extends Action = Action, V extends IActio
       });
   }
 
+  private showFirstN(visibleItemsCount: number) {
+    let leftItemsToShow = visibleItemsCount;

```

Comment:
Is that by design that `visibleItemsCount` is unused? It is possible to use `visibleItemsCount`  instead `leftItemsToShow`